sonar-project.properties

```diff
@@ -3,4 +3,7 @@ sonar.projectKey=ohsome-quality-analyst
 sonar.projectName=ohsome quality analyst
 
 # disable PL/SQL
-sonar.plsql.file.suffixes=""
\ No newline at end of file
+sonar.plsql.file.suffixes=""
+
+# exclude non-API worker files from coverage report
+sonar.coverage.exclusions=database/**,website/**,workers/scripts/**,workers/tests/**
```

Comment:
Should we also include the docs/ directory? There are no code files so maybe that is irrelevant.

Reply:
Unfortunately, I cannot do something like this: `!workers/ohsome_quality_analyst/**`. Therefore, I only added directories which are actually checked with sonarcloud. I did not add all files, since this would have been a long list. But if you want, I can add `docs/**`.

Reply:
Yes thats sounds fine to me. Lets do in the future if it is checked by sonar lint